dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -684,7 +687,9 @@ def _build_package_with_docker(build_distribution_format: str):
             _build_package_with_docker(
                 build_distribution_format=distribution_format,
             )
-    get_console().print(f"[success]Successfully prepared Airflow {distribution_pretty_name} packages")
+    get_console().print(
```

Comment:
This is just to change logging as `airflowctl` to make it prettier but due to directory structure some parts needs to stay as apache-airflow-ctl for now